src/api/customer/customer-serializer.ts

```diff
@@ -19,6 +21,48 @@ const createCustomerSerializer: CustomerCreationRequestHandler = (req, res, next
     next();
 }
 
+const findCustomerSerializer: CustomersFindRequestHandler = (req, res, next) => {
+    
+    const { customerFind } = res.locals;
+    if(customerFind != undefined){
```

Comment:
Poderia simplificar.
Pois você tem certeza que o objeto vai ser sempre uma instância de customer ou undefined.

```suggestion
    if (customerFind) {
```